tdrs-frontend/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "tdrs-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@trussworks/react-uswds": "^1.8.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.0",
+    "typescript": "^3.9.7"
+  },
+  "scripts": {
```

Comment:
Moved these around a bit to consolidate the similarly named scripts. 

Renamed some of the scripts to keep consistent pattern

Extracted `HOST=localhost BROWSER=none` into it's own `start:ci` script to keep it DRY and keep the `test:accessibility` and `cy:run:ci` clean and easy to read. 

restored the `test` script to its original, which just runs Jest since the `test:stuff` wasn't being used.